scripts/azureml-assets/azureml/assets/release/model_publish_utils/utils.py

```diff
@@ -67,15 +67,27 @@ def _download_git_model(self) -> bool:
             if commit_exists != 0:
                 # TODO: Error handling incase of incorrect commit hash.
                 logger.log_warning("Commit hash doesn't exist. Using model from latest HEAD.")
-        git_path = os.path.join(self.model_dir, '.git')
+        git_path = os.path.join(self.model_dir, ".git")
         shutil.rmtree(git_path, onerror=_onerror)
         return True
 
+    def _download_azure_artifacts(self) -> bool:
+        """Download model files from blobstore."""
+        az_copy_cmd = f"azcopy cp --recursive=true {self.model_url} {self.model_dir}"
```

Comment:
> {self.model_dir} [](http://example.com/codeflow?start=67&length=17)

Is this the target registry storage dir?

Reply:
`self.model_url` is target storge dir. `model_dir` is where it is going to download on the agents